targets/UnrealMarketplacePlugin/makeCpp.js

```diff
@@ -660,6 +661,20 @@ function getRequestActions(tabbing, apiCall, isInstanceApi) {
     return "";
 }
 
+function getCustomApiLogic(tabbing, api, apiCall, isInstanceApi) {
+    if (apiCall.name === "ExecuteFunction")
+        return tabbing + "FString localApiServer = PlayFabSettings::GetLocalApiServer();\n"
+            + tabbing + "if (!localApiServer.IsEmpty())\n"
+            + tabbing + "{\n"
+            + tabbing + "    FString endpoint = TEXT(\"" + apiCall.url + "\");\n"
+            + tabbing + "    endpoint.RemoveFromStart(TEXT(\"/\"));\n"
```

Comment:
Url string construction - ditto.

Reply:
Resolving this as per our conversation in person.